Templates/CSharp/Requests/IMethodRequest.cs.tt

```diff
@@ -72,20 +91,186 @@ namespace <#=method.Namespace.GetNamespaceName()#>
         sendParameterHeadersForOverload = "/// <param name=\"cancellationToken\">The <see cref=\"CancellationToken\"/> for the request.</param>";
         sendOverloadParameters = "CancellationToken cancellationToken";
     }
+#>
+
+<# 
+    if(isAction) // POST
```

Comment:
Created templates for each verb supported for the method (action, function, composable function)